test_pro_access.py

```
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+"""
+Test script to verify OpenAI Pro access and available models
+"""
+import openai
+import os
+from typing import List, Dict
```

Flake8 (7.2.0)

[error] 'typing.List' imported but unused

(F401)